src/bundle/Resources/public/scss/_dropdown.scss

```diff
@@ -456,6 +455,18 @@
                 padding: 0 calculateRem(24px) 0 calculateRem(16px);
             }
         }
+
+        &.ibexa-dropdown--disabled {
+            .ibexa-dropdown__selection-info {
+                background-color: transparent;
+                color: $ibexa-color-dark-300;
+
+                &::before,
+                &::after {
```

Comment:
Before and after do not need content: '' anymore?

Reply:
`content: ''` was specified in that file: https://github.com/ibexa/admin-ui/blob/7abf88ed55f3d594276a40866acbf302d5eca051/src/bundle/Resources/public/scss/_dropdown.scss#L40
here we only override `background-color`.